plugins/kinet/KiNetPort.h

```diff
@@ -42,17 +43,54 @@ class KiNetOutputPort: public BasicOutputPort {
         m_target(target) {
   }
 
+  virtual bool WriteDMX(const DmxBuffer &buffer, OLA_UNUSED uint8_t priority) = 0;
+
+  virtual std::string Description() const = 0;
+
+ protected:
+  KiNetNode *m_node;
+  const ola::network::IPV4Address m_target;
+};
+
+
+class KiNetDmxOutOutputPort: public KiNetOutputPort {
+ public:
+  KiNetDmxOutOutputPort(KiNetDevice *device,
+                        const ola::network::IPV4Address &target,
+                        KiNetNode *node)
+      : KiNetOutputPort(device, target, node, 0) {
+  }
+
   bool WriteDMX(const DmxBuffer &buffer, OLA_UNUSED uint8_t priority) {
     return m_node->SendDMX(m_target, buffer);
   }
 
   std::string Description() const {
-    return "Power Supply: " + m_target.ToString();
+    return "DMX Out Port";
```

Comment:
I think using "port" is confusing on this one. I know that is the OLA terminology, but since "Port Out" is also specific to KINET, maybe instead it could be `DMX Out Mode`?

Reply:
Maybe `DMX Out Mode, Output Port` or something, although it's a bit long.

We can also make the device more explicit which might make some of this redundant.

Reply:
Now changed to something similar, thoughts?

Reply:
Looks good!